Robinhood/Robinhood.py

```diff
@@ -73,11 +57,77 @@ class Robinhood:
     logger = logging.getLogger('Robinhood')
     logger.addHandler(logging.NullHandler())
 
+    from . import exceptions
+
+    class Bounds(Enum):
```

Comment:
Enums moved inside class?  Why?  Won't this break tests since enums are no longer referenceable directly?